person.rb

```diff
@@ -0,0 +1,33 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1...1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age?
+  end
+
+  private
+
+  def of_age?
+    @age >= 18 || @parent_permission
+  end
```

Comment:
- [ ] As per these [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=%40age.-,Private%20method%20of_age%3F%20that%20returns%20true%20if%20%40age%20is%20greater,person%20is%20of%20age%20or%20if%20they%20have%20permission%20from%20parents.,-Create%20class%20Student), Please make the `can_use_services?` method returns true if the person is of age or if `they have permission from parents`, not only if the person of age as you did.
- [ ] For the `of_age?` method only returns true if the person is only 18 years or older.

[hint]: It will be like that.
```ruby
def can_use_services?
    of_age? || @parent_permission
  end

  private

  def of_age?
    @age >= 18
  end
```

Reply:
Hi @Omar-Muhamad , thanks for the correction, it is done.